HLint.hs

```diff
@@ -3,6 +3,8 @@ import "hlint" HLint.Dollar
 import "hlint" HLint.Generalise
 import "hlint" HLint.HLint
 
+ignore "Avoid lambda" = Action.mkRunAction
```

Comment:
just tried this quickly, and failed.  your solution is good, too.  i was just curious.